rbac/rbac/middleware.py

```diff
@@ -230,6 +230,9 @@ def process_request(self, request):  # pylint: disable=R1710
 
             super().process_request(request)
             # We are now in the database context of the tenant
+            if settings.SERVE_FROM_PUBLIC_SCHEMA:
+                connection.set_schema_to_public()
```

Comment:
Can/should we consolidate these two calls so they're the same? https://github.com/RedHatInsights/insights-rbac/pull/589/files#diff-642029e74ba2ca49cb6da38bfbfae49d94851a96e507abf8a9c5cef1dd000ec3R78